src/Microsoft.Health.Fhir.TemplateManagement/Client/ACRClient.cs

```diff
@@ -43,7 +51,7 @@ public async Task<Stream> PullBlobAsStreamAcync(string imageName, string digest,
             try
             {
                 cancellationToken.ThrowIfCancellationRequested();
-                rawStream = await _client.Blob.GetAsync(imageName, digest, cancellationToken);
+                rawStream = await _retryPolicy.ExecuteAsync(async () => await _client.Blob.GetAsync(imageName, digest, cancellationToken));
```

Comment:
also add cancellationToken into  ExecuteAsync

Reply:
Thanks, fixed it.